rs/registry/canister/src/mutations/do_remove_node_operators.rs

```diff
@@ -73,3 +67,50 @@ impl Registry {
         }
     }
 }
+
+/// The payload of a request to remove Node Operator records from the Registry
+#[derive(Clone, Eq, PartialEq, CandidType, Deserialize, Message, Serialize, Hash)]
+pub struct RemoveNodeOperatorsPayload {
+    #[prost(bytes = "vec", repeated, tag = "1")]
+    pub node_operators_to_remove: Vec<Vec<u8>>,
+
+    // In protobuf (specifically proto3), "repeated" fields are not truly "optional"
```

Comment:
Given we can drop protobuf support for this type (since it's only API and not internal storage), could this be reformulated in the way you want it to be in candid?

Reply:
These types are not used internally (i.e. stored as registry values), so they don't need protobuf.